siriuspy/siriuspy/orbintlk/main.py

```diff
@@ -1233,8 +1233,9 @@ def _check_configs(self):
             value = _updt_bit(value, 8, not okg)
         else:
             value += 0b11 << 7
-        # Delta Interlock Redundancy EVR 
-        if self._evrdelta_dev.connected:
+        # Delta Interlock Redundancy EVR
+        dev = self._evrdelta_dev
+        if dev.connected:
```

Comment:
I didn't understand this bug fix.

Reply:
The bug is in fact in the following rows, when I use the object `dev`. Once this was not redefined, the statement was trying to read a PV from a previous defined `dev`, and not from `self._evrdelta_dev`